### ISSUE TITLE
Lack of documentation or descriptive coding conventions

### ISSUE THREAD

**Issue by munsanje** [siavashk/pycpd#24]
This issue was raised in #12 and then closed, despite the actual issue persisting. Think it would be valuable to leave this open so that potential contributors know that it is an open problem.

Would also be useful if the variable names were descriptive, so that the burden of explicitly documenting the code is lessened. Variables like `B` and `t` do nothing to explain the functionality and hinder understanding.

**Comment by siavashk** [comment]
You are right. I wrote `pycpd` over the weekend as part of an invited talk and to be honest, I did not think anyone other than the people who attended the talk would be interested.

I tried to have variables mirror their counterparts in the original [paper](https://arxiv.org/abs/0905.2635) as this package was meant for educational use. So I will probably not rename (`B`, `t`) to something like (`affine_transformation_matrix`, `translation_vector`). What I will do is point out the connection in comments.

If you have other suggestions on how the package can be improved, you are more than welcome to post them here, or better yet, fork the repository and submit a pull request.
